core/core.go

```diff
@@ -108,6 +107,26 @@ var (
 		},
 	}
 
+	// allowedCrossUserMethods are methods allowed to be called by users who do not own the target thread.
+	allowedCrossUserMethods = []string{
+		"/threads.pb.API/Save",
+		"/threads.pb.API/Has",
+		"/threads.pb.API/Find",
+		"/threads.pb.API/FindByID",
+		"/threads.pb.API/ReadTransaction",
+		"/threads.pb.API/WriteTransaction",
+		"/threads.pb.API/Listen",
+		"/api.buckets.pb.APIService/Root",
+		"/api.buckets.pb.APIService/Links",
+		"/api.buckets.pb.APIService/ListPath",
+		"/api.buckets.pb.APIService/PushPath",
+		"/api.buckets.pb.APIService/PullPath",
+		"/api.buckets.pb.APIService/SetPath",
+		"/api.buckets.pb.APIService/RemovePath",
+		"/api.buckets.pb.APIService/GetPathAccessRoles",
```

Comment:
Should this and the one below be `PullPathAccessRoles` and `PushPathAccessRoles` based on the client code above?

Reply:
Yep, good catch